src/containers/DailyReportPage/index.js

```diff
@@ -0,0 +1,25 @@
+import React, { Component } from 'react';
+import { Route, Switch } from 'react-router-dom';
+import CreateDailyReportContainer from './CreateDailyReportContainer';
+import UpdateDailyReportContainer from './UpdateDailyReportContainer';
+import DailyReportDetailContainer from './DailyReportDetailContainer';
+import DailyReportContainer from './DailyReportContainer';
+import NoMatch from '../../utils/NoMatch';
+
+class DailyReportPage extends Component {
+  render() {
+    return (
+      <div>
```

Comment:
FYI : 
I cloned your forked repo and deleting this `<div>` solved layout problem.
I will fix this after this pull request is merged 👍